README.md

```diff
@@ -75,25 +78,22 @@ following design principles:
      job will abandon as soon as possible and the Forget phase will not take
      place. The Forget Phase takes the list of objects returned from the Find
      phase, and deletes only the relevant rows in those objects.
-   - **Optimised for Parquet:** The process of separating the two phases, as
-     opposite of processing the read/write on an object basis, is another cost
-     optimisation technique leveraging columnar dense formats such as Parquet
-     allow to efficiently find matches by reading only relevant columns instead
-     of full content from all columns. By processing only the relevant columns
-     during the Find and delegating the queries to a serverless managed service
-     such as Amazon Athena, we can then process full rows only during the Forget
-     phase, restricting the most expensive operation to only the relevant
-     objects.
+   - **Optimised for Parquet:** The split phase approach optimises scanning for
+     columnar dense formats such as Parquet. The Find phase only retrieves and
+     processes the data for relevant columns when determining which S3 objects
+     need to be processed in the Forget phase. This approach can have
+     significant cost savings when operating on large data lakes with sparse
+     matches.
    - **Serverless:** Where possible, the solution only uses Serverless
-     components, in order to avoid paying for idle resources. All the components
-     for Web UI, API and Deletion Jobs are Serverless (for more information
-     consult the [Cost Overview guide](docs/COST_OVERVIEW.md)).
+     components to avoid paying for idle resources. All the components for Web
```

Comment:
```suggestion
     components to avoid costs for idle resources. All the components for Web
```

In the current form it could sound like _the solution_ is avoiding paying.

Reply:
Done